usr/share/rear/conf/default.conf

```diff
@@ -1135,14 +1135,14 @@ DUPLY_PROFILE=""
 # BACKUP_DUPLICITY_OPTIONS="--volsize 100"
 #
 # directories NOT to backup
-# NOTE: for mountpoints to restore use MOUNTPOINTS_TO_RESTORE (extra)
+# For mountpoints that are not in the backup to restore use DIRECTORIES_TO_CREATE
 # Mountpoints for proc (e.g. /proc and /var/lib/ntp/proc) and /sys MUST given -
 # if not duplicity will fail!
 # BACKUP_DUPLICITY_EXCLUDE=( '/proc' '/sys' '/run' '/var/lib/ntp/proc' "$HOME/.cache" '/tmp' '/var/tmp' '/app' '/var/app' )
 #
 # Mountpoints to restore
 # if defined, used by restore/default/900_create_missing_directories.sh
-# MOUNTPOINTS_TO_RESTORE="proc sys run tmp dev/pts dev/shm app app/rear var/app"
+# DIRECTORIES_TO_CREATE="proc sys run tmp var/tmp dev/pts dev/shm app app/rear var/app"
```

Comment:
If the other directories are hard coded in the script then let's take them out of the example to make the example more obvious, that it is meant for additional directories.

Reply:
I agree - bad example - I will enhance it.